src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.js

```diff
@@ -25,45 +25,17 @@ function hasSecondaryHighlighted({ hoveredItem, breakpoint }) {
   return breakpoint.id == getLocationKey(hoveredItem.location);
 }
 
-export function getLeftPercentOffset({ point, timelineNode, zoomRegion, markerWidth }) {
-  const timelineWidth = timelineNode.getBoundingClientRect().width;
-
-  const startTime = zoomRegion.startTime;
-  const endTime = zoomRegion.endTime;
-
-  // Percent offset values are [0, 100].
-  const unadjustedLeftPercentOffset = ((point.time - startTime) / (endTime - startTime)) * 100;
-  const unadjustedLeftPixelOffset = (unadjustedLeftPercentOffset / 100) * timelineWidth;
-  const leftPixelOffset = unadjustedLeftPixelOffset - markerWidth / 2;
-  const leftPercentOffset = (leftPixelOffset * 100) / timelineWidth;
-  return leftPercentOffset;
-}
```

Comment:
don't know how you're getting rid of this math, but i like it

Reply:
[That's how](https://github.com/hbenl/webreplay-devtools/blob/55ffa9c433bd82eb2e31faa85ff5b6fe6c06c904/src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.js#L67). CSS is awesome.